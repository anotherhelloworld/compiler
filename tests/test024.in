var
var1, double1: double;