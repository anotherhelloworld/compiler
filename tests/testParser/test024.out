Illegal expression. Line 1 Column 2
